tools/lou_checkyaml.c

```diff
@@ -180,6 +221,32 @@ read_table(yaml_event_t *start_event, yaml_parser_t *parser) {
 			strcat(table, (const char *)event.data.scalar.value);
 		}
 		yaml_event_delete(&event);
+	} else {  // event.type == YAML_MAPPING_START_EVENT
+		char *query;
+		const char *table_file_name_check = NULL;
+		yaml_event_delete(&event);
+		query = read_table_query(parser, &table_file_name_check);
+		table = lou_findTable(query);
+		free(query);
+		if (!table) exit(EXIT_FAILURE);
```

Comment:
Shouldn't we print an error message here before dying?

Reply:
Yes you're right.